src/server/routes/consents.ts

```diff
@@ -152,15 +162,20 @@ export const consentPages: ConsentPage[] = [
     page: '/newsletters'.slice(1),
     path: '/consents/newsletters',
     pageTitle: CONSENTS_PAGES.NEWSLETTERS,
-    read: async (ip, sc_gu_u, geo) => ({
-      page: '/newsletters'.slice(1),
-      newsletters: await getUserNewsletterSubscriptions(
-        NewsletterMap.get(geo) as string[],
-        ip,
-        sc_gu_u,
-      ),
-      previousPage: '/communication'.slice(1),
-    }),
+    read: async (ip, sc_gu_u, geo, newsLettersTestActive) => {
+      const newsletterMap = newsLettersTestActive
+        ? TestNewsletterMap
+        : NewsletterMap;
+      return {
+        page: '/newsletters'.slice(1),
```

Comment:
realise you didn't add this, but I'm confused by our pattern here of doing `.slice(1)` instead of just putting `newsletters` 🤔  we do it below as well

Reply:
Actually it looks like a hangover from when we did the type safety on the `Routes` object a couple of months back https://github.com/guardian/gateway/pull/1224 before we'd take the `Routes` enum, the route, and remove the `/` (`Routes.CONSENTS_COMMUNICATION.slice(1)`), however since we're not using enums anymore it does look a bit strange.

Since they're just `string` types now, i think you can just remove the `/` and `.slice(1)`. Ideally we could make these type safe too, specifically the `page` and `previousPage` properties as the are related to the `path`.

Reply:
Yeah I noticed this and agree it was a bit strange. Didn't touch as it wasn't neccessarily related to my changes, but I can clean this up.

Reply:
I think we could define the paths with template literal types, e.g.

```typescript
type ConsentPath = 'communication' | 'newsletters' | 'data' | 'review';
type RoutePaths = `/consents/${ConsentPath}` | '/consents/:page';
```

But might be best to leave this for another PR.